### PR TITLE
please check index view file 

### PR DESCRIPTION
**Pull request by prgrmr-yn**
routes is working and its working

### CODE REVIEW

**Comment by stephen-w-choo** [comment]
rails generate migration add_name_to_powerbanks name:string

File: app/views/powerbanks/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<h1>Available powerbanks</h1>
+
+<ul>
+  <% @powerbanks.each do |powerbank| %>
+    <li><%= powerbank.description %></li>
+    <li><%= powerbank.price %></li>
+    <li><%= powerbank.accessories %></li>
```
Comment (demisaurusRex) [inline app/views/powerbanks/index.html.erb:10]: We will need to add in a name separate to the description as we will need to pull the name for the dashboard and index pages
There will also need to be an indication of availability when they are listed

Comment (prgrmr-yn) [reply app/views/powerbanks/index.html.erb:10]: Thanks @demisaurusRex 
I can do that on thursday and then we can review again

**Review by demisaurusRex — APPROVED** [review]
All updated and migration added

**Pull request merged**